content/en/continuous_integration/intelligent_test_runner/python.md

```diff
@@ -57,8 +57,61 @@ After completing setup, run your tests as you normally do:
 DD_ENV=ci DD_SERVICE=my-python-app DD_CIVISIBILITY_ITR_ENABLED=true pytest --ddtrace
 {{< /code-block >}}
 
+## Preventing the Intelligent Test Runner from skipping specific tests or modules
+
+You can override the Intelligent Test Runner's behavior and prevent specific tests or modules from being skipped. These tests are referred to as unskippable tests.
+
+### Why make tests unskippable?
+
+The Intelligent Test Runner uses coverage data to determine whether or not tests should be skipped. In some cases, coverage data may not be sufficient to make this determination.
```

Comment:
To me `coverage data` seems a little big vague, in the sense that it is not obvious that ITR only considers changes in the code. I'd suggest making it a bit more explicit, e.g. `code coverage data` at least.

Reply:
Updated the text to say "code coverage data" isntead.